vpn/README.md

```diff
@@ -34,16 +34,21 @@ Follow these instructions to setup and connect. Once connected to the VPN, traff
 1. To configure `ssoca` for the first time:
 
     ```
-    ssoca -e bosh env set https://vpn-bosh.ci.cf-app.com
+    ssoca -e bosh env add https://vpn-bosh.ci.cf-app.com
+    ```
+    
+2. Once your environment has been added, authenticate:
```

Comment:
You should be prompted for authentication automatically, so this step shouldn't really be necessary. The server was upgraded recently which may have caused manual login to be required; you may want to download the latest client.